packages/neo-one-server-client/src/Client.js

```diff
@@ -151,6 +173,40 @@ export default class Client {
     });
   }
 
+  async createResource({
+    plugin,
+    resourceType,
+    name,
+    options,
+    cancel$,
+  }: {|
+    plugin: string,
+    resourceType: string,
+    name: string,
+    options: Object,
+    cancel$: Observable<void>,
+  |}): Promise<?BaseResource> {
+    const response = await this.createResource$({
+      plugin,
+      resourceType,
+      name,
+      options,
+      cancel$,
+    }).toPromise();
+
+    const error = getTasksError(response.tasks);
+    if (error != null) {
+      throw new Error(error);
+    }
+
+    return this.getResource({
```

Comment:
Should probably `await` this and then enforce it's not null so that `createResource` can return a non-null value.